src/app/api/gql/dataset-collaboration/dataset-search-collaborator.graphql

```diff
@@ -0,0 +1,9 @@
+query datasetSearchCollaborator($query: String!, $filters: LookupFilters!, $page: Int, $perPage: Int) {
```

Comment:
I don't think it is anyhow related to dataset, and should rather be a part of Search feature.

Reply:
made a part of Search feature